docs/learning/backend/CodeService.md

```diff
@@ -125,12 +124,64 @@ Generally it is a good idea to reserve a group of codes together rather than one
 - `reserveCode` to reserve a single code
 - `reserveCodes` to reserve a set of codes
 
+Here are the steps to follow.
+
+The identity of the code service and information about how to connect are stored in the iModel persistently. When you open a BriefcaseDb for that iModel, it will read the configuration data from the iModel and automatically try to connect to the code service. When an app opens a briefcase, the briefcase will connect to the code service using the connection information stored in the iModel. The briefcase `.codeService` property will refer to the connected code service client, and apps must use that to reserve, update, and query codes and CodeSpecs.
```

Comment:
```suggestion
The identity of the code service and information about how to connect are stored in the iModel persistently. When you open a BriefcaseDb for that iModel, it will read the configuration data from the iModel and automatically try to connect to the code service. When an app opens a briefcase for editing, the briefcase will connect to the code service using the connection information stored in the iModel and the user's credentials. The briefcase `.codeService` property will refer to the connected CodeService client, and apps must use that to reserve, update, and query codes and CodeSpecs.
```

Reply:
done